daemon/src/wallet.rs

```diff
@@ -607,4 +619,51 @@ mod tests {
             .unwrap()
             .expect("single UTXO to be available after unlocking it");
     }
+
+    #[tokio::test]
+    async fn given_margin_less_balance_then_insufficient_funds_error_when_building_party_params() {
```

Comment:
You only test the negative case here. Shouldn't we add a positive one as well?

Reply:
The positive one is implicitly tested in the already existing tests.